screens/AddHotspotScreen.tsx

```diff
@@ -389,7 +403,18 @@ export const AddHotspotScreen = () => {
               labelStyle={styles.saveButtonLabel}
               icon="check"
               mode="contained"
-              onPress={() => alert('save')}
+              onPress={() =>
+                alert(
+                  JSON.stringify({
+                    name: name,
+                    status: status,
+                    remarks: remarks,
+                    contactPerson: contactPerson,
+                    contactPersonPhone: contactPersonPhone,
+                    volunteer: volunteer,
+                  })
```

Comment:
in javascript and typescript if the names for the key and the value ar the same then you can omit specifying both.
I. e.: 

```
{
  name: name,
  status: status,
  remarks: remarks,
  contactPerson: contactPerson,
  contactPersonPhone: contactPersonPhone,
  volunteer: volunteer,
}
```
is the same as 
```
{
  name,
  status,
  remarks,
  contactPerson,
  contactPersonPhone,
  volunteer,
}
```

And the second variant is preferred because it's less verbose.

Reply:
done.